api/mesh/v1alpha1/dataplane_helpers.go

```diff
@@ -16,6 +16,9 @@ const (
 	KubeNamespaceTag = "k8s.kuma.io/namespace"
 	KubeServiceTag   = "k8s.kuma.io/service-name"
 	KubePortTag      = "k8s.kuma.io/service-port"
+	// KDSSyncLabel a label that controls properties of the KDS sync.
+	// currently only disabled/enabled is supported
+	KDSSyncLabel = "kuma.io/kds-sync"
```

Comment:
Don't we need a validator that forbids setting this label for some resources? Example: `Mesh` object

Reply:
We could though I think this label is niche enough that people using it should know what they are doing.

Also you could imagine having a mesh on global and don't want zones to get it yet.